### PR TITLE
Add Image Definition

### PR DESCRIPTION
**Pull request by Renegade-Master**
- .containerignore
  - Add all files to ignore list, and exclude necessary files.

- .github/workflows/verify_commit.yml
  - Add stage to verification Job to validate that the Image can successfully build.

- README.md
  - Added Podman as Docker alternative.

- image.brnvs.Containerfile
  - Add Image build instructions file.

### CODE REVIEW

**Comment by Renegade-Master** [comment]
@Ziggiyzoo 
I split the Workflow Jobs up a bit, but this does mean that I need you to change the "required" checks in the settings again. This should hopefully be the last time, at least for a while.

File: .github/workflows/verify_commit.yml [hunk 1]
```diff
@@ -14,12 +14,12 @@ defaults:
...
 jobs:
-  lint_test_run:
-    name: "Lint, Test, and Run the Project"
+  lint_application:
+    name: "Lint the Project"
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ "3.8", "3.9", "3.10" ]
+        python-version: [ "3.9" ]
```
Comment (Ziggiyzoo) [inline .github/workflows/verify_commit.yml:22]: We are only doing a Lint on 3.9?

File: src/logic/slash_logic.py [hunk 2]
```diff
@@ -4,17 +4,12 @@
 
 from src.logic import resources_logic
 
-# pylint: disable=too-few-public-methods
-class BrvnsLogic():
+
+def signup_string(author_name: str):
```
Comment (Ziggiyzoo) [inline src/logic/slash_logic.py:8]: Doing this just means we have to change it back into a class once there are more methods being used by the slash commands..

Comment (Ziggiyzoo) [reply src/logic/slash_logic.py:8]: Or should it just be a file filled with methods for all of the logic? I guess as it doesn't need anything from self?

Comment (Renegade-Master) [reply src/logic/slash_logic.py:8]: Yeah. Making these independent makes them easier to test. The BRVNS Bot Class can manage state, passing it here when necessary.

**Comment by Ziggiyzoo** [comment]
Looks good!

**Review by Ziggiyzoo — APPROVED** [review]
Ziggiy's stamp of approval!

**Pull request merged**